pkg-tests.sh

```diff
@@ -4,100 +4,10 @@
 
 set -e
 
+# TODO: why do we set TERM here? Perhaps due to https://github.com/gap-system/gap/issues/1617 ?
+# -> Ask Alex and Thomas about this, and either remove it, or else keep it but then document it
```

Comment:
This was present in the `.travis.yml` files of the old `gap-infra` repositories, it was set before calling this script. I moved the declaration of the variable here, so I'm the one who put it here. But I don't know what it does or if it is still necessary.

Reply:
yes, https://github.com/gap-system/gap/issues/1617 seems relevant, perhaps similar issue was on Travis.